docs/_Docs_ReputationMining.md

```diff
@@ -4,4 +4,55 @@ section: Docs
+
+The process works on an “innocent until proven guilty” principle: Although it’s impossible for a valid hash to be proven true on-chain, it’s relatively straightforward for an invalid hash to be proven false. In the event that two submitted hashes differ, the false one can be fleshed out through an automated challenge process *on-chain* that will eventually zero in on the exact on-chain events that differ between the two submitted hashes.
+
+### Events that affect Reputation scores.
+One of the features of reputation that distinguish it from a token is that it cannot be transferred between accounts through a voluntary transaction.
+
+Reputation within a specific colony can only be earned and lost by completing tasks within the colony, through the [objections and disputes mechanism](/colonynetwork/docs-objections-and-disputes/), and in the case of the Meta Colony, through participation in the reputation mining process.
+
+## The Reputation Mining Cycle
+
+
+### Reputation Root Hashes
+Using a [Patricia Tree](https://github.com/ethereum/wiki/wiki/Patricia-Tree), all updates to the global state of reputation are maintained on-chain with a single "Reputation Root Hash".
+
+The Reputation Root Hash is effectively a fingerprint for the network state (of reputation). It must be calculated off-chain, but it represents the collection of all reputation-updating events that have occured on the network, and even a single discrepancy in the complete history of reputational updates will result in a different Root Hash.  
+
+Miners are incentivized to maintain consensus over the state of reputation by repeatedly calculating the Reputation Root Hash and submitting it to the network.
+
+### Submissions
+Each cycle begins with a submission window, in which all miners compete to submit a correct Root Hash to the network.
+
+A miner is eligible to submit a new Root Hash only if they stake an amount of CLNY. The amount of CLNY staked determines the number of times a miner can submit a Root Hash to the network during each submission window, thus the greater the stake, the greater the likelihood that a miner will receive rewards from a successful submission.
+
+### Challenges
+The challenge-response protocol is described in section 7.5.1 of the [Colony whitepaper](https://colony.io/whitepaper.pdf).
+
+Leaving the exact mechanism aside, the general process can be summarized as follows:
+
+If two different root hashes are submitted, one must be false, and the miner that submitted an invalid hash must be punished.
+
+In whichever submission is false, there must be one or more reputation update that is not consistent with the global state of reputation up until the current submission window.
+
+Miners can submit a justification to the network that show how a reputation update results in a Root Hash consistent with the network state right before this submission window, and they can do so for each mining cycle going back to the network's beginning.
```

Comment:
'that shows'

The justification hash essentially commits the miner to not change their 'working' during the dispute resolution process in a way that can be verified on chain. It only represents the 'working' during the current mining window; it contains no proof for any previous windows. They could in theory be produced, but there's no real reason to.